### ISSUE TITLE
Button is Not Handling Overflow Wrapping

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#398]
# Description
```
Please provide a short description of what you have encountered below.
```
Due to many overflapping properties like `overflow-wrap` and `word-break` being used together.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
Fixed version should behave like this:

![button-after](https://github.com/ZORALab/Hestia/assets/7259572/76fccde5-c427-430d-9e7d-4376e9030edb)

# Current Behavior
```
Please specify the current behavior of software below.
```

![button-before](https://github.com/ZORALab/Hestia/assets/7259572/265b74c9-c88c-4d3d-8257-f06c38a67aec)

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
1. Make a super long text labelled button.
2. View in Galaxy Fold form factor.
3. Observe the pattern.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
fixed in 746a8c22fc55d903d2c0f1b906c6af4d396b0eeb

**Comment by hollowaykeanho** [comment]
Released in v1.2.1.
